MiniApp/Classes/ui/MiniAppViewController.swift

```diff
@@ -191,31 +191,27 @@ public class MiniAppViewController: UIViewController {
 
     @objc
     public func backPressed() {
-        if delegate == nil {
-            navBarDelegate?.miniAppNavigationBar(didTriggerAction: .back)
-        } else {
-            delegate?.miniApp(self, shouldExecute: .back)
-        }
```

Comment:
@laubengaier I removed these lines because, I don't see any reason why we call `delegate?.miniApp(self, shouldExecute: .back)` here. Because the host app neither returns anything (Bool value to execute or not) nor host app doesn't have instance of `navigationBarDelegate` to perform the trigger to go `.back` & `.forward`.

Checking with you here so that removing this won't impact anything on the MALI side.

Reply:
@rleojoseph if I remember well, when I created the navigation delegates for the first time I gave the ability to host app developers to have their own navigation logic, so the possibility to have no navigation bar. But as @laubengaier  made improvements to it maybe now we don't need this anymore

Reply:
@Climbatize , we still can use your delegates as intended. But these delegates are created/meant for a different reason I guess. Lets hear from @laubengaier

Reply:
@leorjoseph yeah as far as I remember the purpose was to allow to override the back and forward actions but if the state is correctly passed and can be accessed via a delegate I see no issue. 
Just a question it's not necessary to provide a (navigation) view for the navigationDelegate to work right?

Reply:
> > lgtm just a comment and a question to the discussion
> 
> @laubengaier Thanks. I don't see the question, what is it ?

damn, must have been accidentally deleted.
If I remember correctly this behavior was indeed as Pierre described to let the host app have their own navigation logic or react to forward / backward changes. If the state is correctly passed through the navigation delegate I think it's fine.
The question was if the navigation delegate is working without the (navigation) view? I remember briefly that something was not working as expected.
_But anyways I have implemented a workaround for the issue we had so shouldn't be critical_

Reply:
@laubengaier 
We still support that I believe. Host app can still receive them and can control whether to go back or forward. 
And like you said, yeah there is a problem with `MiniAppNavigationDelegate` here. 
When you create & launch the MiniApp using `MiniAppUI.shared().launch()` you still have the `MiniAppNavigationDelegate` referred/infered to Host app ViewController, and it works like below,

observeWebView()  	- SDK
	|
	|
miniAppNavigationCanGo(back: Bool, forward: Bool) {}.	- Hostapp
	|
	|
 refreshNavigationBarButtons(backButtonEnabled: Bool, forwardButtonEnabled: Bool) {} - SDK UI